man/ocaml.1

```diff
@@ -190,6 +195,21 @@ interactive session.
 .B \-strict\-sequence
 Force the left-hand part of each sequence to have type unit.
 .TP
+.B \-no-strict\-sequence
+Left-hand part of a sequence need not have type unit.
+This is the default.
+.TP
+.B \-strict\-formats
+Reject invalid formats that were accepted in legacy format
+implementations. You should use this flag to detect and fix
+such invalid formats, as they will be rejected by future
+OCaml versions.
+.TP
+.B \-no-strict\-formats
+Do no reject invalid formats that were accepted in legacy format
+implementations.
+This is the default.
```

Comment:
Main options should be sorted by alphabetical order, thus `strict-format` should be just before `strict-sequence`.